updatecli/updatecli.d/awscli.yml

```diff
@@ -0,0 +1,55 @@
+---
+name: Bump the `aws` CLI version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  lastReleaseVersion:
+    kind: githubrelease
+    name: Get the latest `aws` CLI` version
+    spec:
+      owner: "aws"
+      repository: "aws-cli"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionfilter:
+        kind: semver
+        pattern: ~2 # Use the 2.x.y version
+
+conditions:
+  ## Only check for chocolatey package: source installer from AWS is always faster than chocolatey to publish
+  checkForChocolateyPackage:
+    kind: shell
+    disablesourceinput: true # Do not pass source as argument to the command line
+    spec:
+      command: curl https://community.chocolatey.org/packages/awscli/{{ source "lastReleaseVersion" }} --silent --show-error --location --fail --output /dev/null
+
+targets:
+  updateVersion:
+    name: Update the `aws` CLI version in the provision-env.yml file
+    sourceid: lastReleaseVersion
+    kind: yaml
+    spec:
+      file: "provisioning/tools-versions.yml"
+      key: "awscli_version"
+    scmid: default
+
+pullrequests:
+  default:
+    kind: github
+    scmid: default
+    title: Bump the `aws` CLI version to {{ source "lastReleaseVersion" }}
+    spec:
+      labels:
+        - enhancement
```

Comment:
Just noticed it's "enhancement" here instead of "dependencies" like in the other repos.
WDYT if I open another PR to align this repo with the others?

Reply:
The initial idea was to separate "dependencies" (such as packer plugins, github actions) from "image dependency changes" (such as git version update for instance).

"Enhancement" means "should trigger a semantic version bump on the minor digit" while dependency is only about the tooling.

That is something that bother me a lot on the other repositories. But having a repo different than the others is clearly worse: go for your PR!
WDYT if in your PR you change the "depencencies" labels to "chore" (for the packer plugins) and the "enhancement" to "dependencies (and update the release-drafter config accordingly)?